iac-extensions/docker/src/main/java/org/sonar/iac/docker/parser/grammar/DockerGrammar.java

```diff
@@ -190,6 +190,13 @@ public EnvTree ENV() {
     );
   }
 
+  public UserTree USER() {
+    return b.<UserTree>nonterminal(DockerLexicalGrammar.USER).is(
+      f.user(b.token(DockerKeyword.USER), b.token(DockerLexicalGrammar.USER_NAME),
+        b.optional(b.token(Punctuator.COLON)), b.optional(b.token(DockerLexicalGrammar.USER_GROUP)))
```

Comment:
See the comment above. Maybe a tuple makes more sense. Right now, a single colon without group would also be allowed.

Reply:
I put the tuple in place and added a test case for it.